### PR TITLE
feat(docs): FRMW-196 - Migrate rxSortableColumn docs

### PR DESCRIPTION
**Pull request by bry4n**
JIRA: https://jira.rax.io/browse/FRMW-196

* [x] Dev LGTM
* [x] Dev LGTM

### CODE REVIEW

File: src/rxSortableColumn/rxSortableColumn.js [hunk 1]
```diff
@@ -24,6 +25,9 @@ angular.module('encore.ui.rxSortableColumn', [])
  * Renders a clickable link in a table heading which will sort the table by
  * the referenced property in ascending or descending order.
  *
+ * The module also provides an optional filter - <code>rxSortEmptyTop</code> -
```
Comment (100stacks) [inline src/rxSortableColumn/rxSortableColumn.js]: Replace `<code>` with backticks-   `<code>rxSortEmptyTop</code>`

Comment (bry4n) [inline src/rxSortableColumn/rxSortableColumn.js]: :+1:

File: src/rxSortableColumn/README.md [hunk 2]
```diff
@@ -1,5 +1,4 @@
 [![unstable](http://badges.github.io/stability-badges/dist/unstable.svg)](http://github.com/badges/stability-badges)
 
-A clickable link in a table heading which will sort the table by the referenced property in ascending or descending order.
-
-The module also provides an optional filter - <code>rxSortEmptyTop</code> - which moves empty values to the top in ascending order, and bottom in descending order.
+A component creates a clickable link in a table heading which will sort the table
+by the referenced property in ascending or descending order.
```
Comment (CITguy) [inline src/rxSortableColumn/README.md]: This description seems to be describing the directive, not the component. Maybe...

```
The rxSortableColumn component provides functionality to sort a table on a single property value.
```

File: src/rxSortableColumn/rxSortableColumn.js [hunk 3]
```diff
@@ -4,7 +4,8 @@
  * @description
  * # rxSortableColumn Component
  *
- * [TBD]
+ * A component creates a clickable link in a table heading which will sort the
+ * table by the referenced property in ascending or descending order.
```
Comment (CITguy) [inline src/rxSortableColumn/rxSortableColumn.js]: Same as README

File: src/rxSortableColumn/rxSortableColumn.js [hunk 4]
```diff
@@ -24,11 +25,17 @@ angular.module('encore.ui.rxSortableColumn', [])
  * Renders a clickable link in a table heading which will sort the table by
  * the referenced property in ascending or descending order.
  *
+ * The module also provides an optional filter `rxSortEmptyTop` which moves empty
+ * values to the top in ascending order, and bottom in descending order.
```
Comment (CITguy) [inline src/rxSortableColumn/rxSortableColumn.js]: This description belongs in the `rxSortEmptyTop` filter description. Please update.

File: src/rxSortableColumn/README.md (hunk 2 above)
Comment (bry4n) [inline src/rxSortableColumn/README.md]: updated

**Comment by CITguy** [comment]
LGTM

**Comment by ssalinas24** [comment]
LGTM

**Pull request merged**